source/imaer-shared/src/main/java/nl/overheid/aerius/geo/shared/Point.java

```diff
@@ -18,23 +18,19 @@
 
 import java.io.Serializable;
 
-import nl.overheid.aerius.shared.MathUtil;
-
 /**
  * Simple point with x and y coordinate.
  */
-public class Point extends Geometry implements Serializable {
+public class Point extends nl.overheid.aerius.shared.domain.v2.geojson.Point implements Serializable {
```

Comment:
Perhaps rename this class to something else, like `SridPoint`? I'd guess we want to prevent to use this class as much as possible if we can, or at least make a clear distinction with the `v2.geojson.Point` version.

Reply:
Or perhaps move the class to `AERIUS-II` repository? The `ShippingNode` class is the only one extending the class in `IMAER-java` as far as I can tell, and I don't think we're using that one anymore?

Reply:
Yes good idea. I've move the class and renamed it there.